style.css

```diff
@@ -1 +1,204 @@
-/*  Add your styling here */
+
+.body{
+    display:grid;
+    grid-template-columns: repeat(4, 1fr);
+    grid-template-rows: auto;  
+    grid-template-areas: 
+    "header header header header"
+    "main main main main" 
+    "footer footer footer footer";
+}
+
+/***HEADER***/
+.header {
+    grid-area: header;
+    background-color:rgb(245, 233, 245) ;
+}
+.homepage-header {
+    display: flex;
+    justify-content: space-between;
+    padding:15px;
+}
+.logo-img {
+    width: 50%;
+    height: 50%;
+    border-radius:80%;
+    object-fit: cover;
+
+}
+.hamburger-logo {
+    width: 60px;
+    height: 40px;
+}
+.header-p {
+    font-family:system-ui;
+    color:rebeccapurple;
+    font-size: 20px;
+    margin: 50px 20px 0 0 ;
+  text-align: right;    
+} 
+.nav-bar {
+    display: none;;
+} 
+/***MAIN**/
+.main {
+    grid-area: main;
+}
+
+/****WELCOME & PARAGRAPH***/
+.homepage-welcome {
+    margin-bottom: 50px;
+} 
+.homepage-welcome h1 {
+    font-size: 50px;
+    color:rebeccapurple;
+    text-shadow: 0 0 3px #FF0000, 0 0 5px #0000FF;
+}
+.homepage-welcome p {
+    font-size:30px;
+}
+
+/***HERO***/
+.homepage-hero img {
+    width:100%; 
+}
+.homepage-hero {
+    margin-bottom: 50px;  
+}
+
+/***PRODUCTS***/
+.homepage-products {
+    display: grid;
+    grid-template-columns:repeat(2,1fr);
+    grid-template-rows: 200px 200px;
+    grid-gap:30px;
+    padding: 0 20px;
+    margin-bottom: 100px; 
+} 
+.products,figcaption {
+    text-align: center;
+    font-size:15px;
+    font-weight: bold;
+    color:black
+}
+.products img {
+    width:100%;
```

Comment:
yes thats correct! having 100% width and height applied to the image(s), those images will then try to fit whatever width of the div that wrap them. I would do exactly the same to achieve responsiveness